packages/ui/core-components/src/lib/unsorted/ui/BigLink.svelte

```diff
@@ -3,10 +3,10 @@
 </script>
 
 <script>
-	export let href;
+	export let link;
```

Comment:
I'm hesitant to make this breaking change. Is there a specific reason we want to?

Reply:
wanted to normalize linking, similar to chart drill down feature. Did not consider breaking current users projects. Reverted back, Thanks for the catch!